Source/WebKit/UIProcess/RemoteLayerTree/ios/RemoteLayerTreeHostIOS.mm

```diff
@@ -152,7 +152,7 @@
 #endif
 
         if (!modelHandledOutOfProcess && m_drawingArea->page() && m_drawingArea->page()->preferences().modelElementEnabled()) {
-            if (auto* model = std::get_if<Ref<Model>>(&properties.additionalData)) {
+            if (std::get_if<Ref<Model>>(&properties.additionalData)) {
```

Comment:
I know you didn't write this, but `std::holds_alternative` seems like the more appropriate and less expensive way to do this (since it wouldn't actually read the stored value)

Reply:
Oh, but actually, @pxlcoder's comment takes priority! 😄